### ISSUE TITLE
interpolation issue

### ISSUE THREAD

**Issue by ArmorDarks** [brigade/scss-lint#243]
Hi

For some reason following code:

``` scss
$test: 'text';

[class~='#{$test}'] { width: 100% !important; }
```

Throws following errors:

```
68: TrailingSemicolon: Declaration should not have a space before the terminating semicolon
68: SpaceAfterPropertyColon: Colon after property should be followed by at least one space
```

While this doesn't throw any errors:

``` scss
$test: 'text';

.#{$test} { width: 100% !important; }
```

My config file:

``` yml
# Default application configuration that all configurations inherit from.
linters:
  BorderZero:
    enabled: true

  ColorKeyword:
    enabled: true

  Comment:
    enabled: false

  DebugStatement:
    enabled: true

  DeclarationOrder:
    enabled: true

  DuplicateProperty:
    enabled: true

  ElsePlacement:
    enabled: true
    style: same_line # or 'new_line'

  EmptyLineBetweenBlocks:
    enabled: true
    ignore_single_line_blocks: true

  EmptyRule:
    enabled: false

  FinalNewline:
    enabled: true
    present: false

  HexLength:
    enabled: true
    style: short # or 'long'

  HexNotation:
    enabled: true
    style: lowercase # or 'uppercase'

  HexValidation:
    enabled: true

  IdWithExtraneousSelector:
    enabled: true

  Indentation:
    enabled: false
    character: tab  # or 'space'
    width: 1

  LeadingZero:
    enabled: true
    style: exclude_zero # or 'include_zero'

  MergeableSelector:
    enabled: true
    force_nesting: true

  NameFormat:
    enabled: true
    convention: BEM # or 'hyphenated_lowercase', or a regex pattern

  PlaceholderInExtend:
    enabled: false

  PropertySortOrder:
    enabled: true
    ignore_unspecified: false

  PropertySpelling:
    enabled: true
    extra_properties: []

  SelectorDepth:
    enabled: true
    max_depth: 3

  SelectorFormat:
    enabled: true
    convention: hyphenated_lowercase # or 'snake_case', or 'camel_case', or a regex pattern

  Shorthand:
    enabled: true

  SingleLinePerProperty:
    enabled: true
    allow_single_line_rule_sets: true

  SingleLinePerSelector:
    enabled: true

  SpaceAfterComma:
    enabled: true

  SpaceAfterPropertyColon:
    enabled: true
    style: at_least_one_space # or 'no_space', or 'one_space', or 'aligned'

  SpaceAfterPropertyName:
    enabled: true

  SpaceBeforeBrace:
    enabled: true
    allow_single_line_padding: true

  SpaceBetweenParens:
    enabled: true
    spaces: 0

  StringQuotes:
    enabled: true
    style: single_quotes # or double_quotes

  TrailingSemicolon:
    enabled: true

  TrailingZero:
    enabled: true

  UnnecessaryMantissa:
    enabled: true

  UnnecessaryParentReference:
    enabled: true

  UrlFormat:
    enabled: true

  UrlQuotes:
    enabled: true

  ZeroUnit:
    enabled: true

  Compass::*:
    enabled: false
```
